### ISSUE TITLE
[Suggestion] Make Advanced Menu More User Friendly

### ISSUE THREAD

**Issue by july8six** [evolus/pencil#411]
* Operating system: Centos 7
* Pencil version: 3.0.4

This is a suggestion to make the advanced menu under settings user friendly.  One doesn't know what the options mean without doing research and most of them are not advanced or complicated options.  

Some ideas
- Instead of having type set as boolean with a true or false, include a checkbox instead. Checked = true, unchecked = false

- Give human readable names to the options

- Remove options in the advanced table that appear in other tabs.  If the options are already listed in another tab, they are just taking up space in the advanced tab for no reason.  Some examples include: object.snapping.enabled, grid.enabled, view.undo.enabled, edit.grid.style, edit.grid.size.

- Put collection options in its own tab with the options in a human readable format.  Some options include setting the visibility and the font settings

- Put options that affect UI into their own tab

- Basically, just break out the options that the average user should understand into their own tab and leave the more advanced options under the advanced tab.

![pencil-advanced-suggestion](https://user-images.githubusercontent.com/25120995/30702447-d423b830-9ea1-11e7-85fe-9363de81ef97.png)
